### PR TITLE
Block SaveWmfAsPngDoesntChangeImageBoundaries

### PR DESCRIPTION
**Pull request by MichalStrehovsky**
People keep running into this locally and I almost spent time investigating this as my own regression.

### CODE REVIEW

**Review by stephentoub — APPROVED** [review]

File: src/libraries/System.Drawing.Common/tests/BitmapTests.cs [hunk 1]
```diff
@@ -790,7 +790,7 @@ public void GetHicon_Disposed_ThrowsArgumentException()
 
         [ConditionalFact(Helpers.IsDrawingSupported)]
         [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "In .NET Framework we use GDI 1.0")]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/72165", typeof(PlatformDetection), nameof(PlatformDetection.IsNativeAot))]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/69471"))]
```
Comment (jkotas) [inline src/libraries/System.Drawing.Common/tests/BitmapTests.cs:793]:
```suggestion
        [ActiveIssue("https://github.com/dotnet/runtime/issues/69471", typeof(PlatformDetection), nameof(PlatformDetection.IsWindows10Version22000OrGreater))]
```

**Comment by MichalStrehovsky** [comment]
> IsWindows10Version22000OrGreater

Is this a Windows 11 check? People are hitting this on Windows 11 (https://github.com/dotnet/runtime/issues/69471#issuecomment-1135152328) and Windows 10 (https://github.com/dotnet/runtime/issues/69471#issuecomment-1188458821) too.

File: src/libraries/System.Drawing.Common/tests/BitmapTests.cs [hunk 2]
```diff
@@ -790,7 +790,7 @@ public void GetHicon_Disposed_ThrowsArgumentException()
 
         [ConditionalFact(Helpers.IsDrawingSupported)]
         [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "In .NET Framework we use GDI 1.0")]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/72165", typeof(PlatformDetection), nameof(PlatformDetection.IsNativeAot))]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/69471", typeof(PlatformDetection), nameof(PlatformDetection.IsWindows10Version22000OrGreater))]
```
Comment (jkotas) [inline src/libraries/System.Drawing.Common/tests/BitmapTests.cs:793]:
```suggestion
        [ActiveIssue("https://github.com/dotnet/runtime/issues/69471")]
```

**Comment by jkotas** [comment]
Ok, I thought this is Win11 specific. It used to work on Win 10 at one point.

**Pull request merged**